app/components/search/SearchMap/SearchMap.tsx

```diff
@@ -25,19 +28,35 @@ export const SearchMap = ({ hits, hitsPerPage, page }: {
             key: config.GOOGLE_API_KEY,
           }}
           defaultCenter={{ lat, lng }}
-          defaultZoom={15}
+          defaultZoom={14}
+          onGoogleApiLoaded={({ map }) => {
+            // SetMapObject shares the Google Map object across parent/sibliing components
+            // so that they can adjustments to markers, coordinates, layout, etc.,
+            setMapObject(map);
+          }}
           options={createMapOptions}
         >
           <UserLocationMarker lat={lat} lng={lng} key={1} />
-          { hits.reduce((markers, hit, i) => {
-            const hitNumber = `${page * hitsPerPage + i + 1}`;
+          { hits.reduce((markers, hit, index) => {
             // Add a marker for each address of each hit
-            hit.addresses?.forEach((addr: any) => {
+            hit.addresses?.forEach((addr: any, i: number) => {
+              const appendAddressLabel = () => {
+                // Appends letter to index number if there are multiple addresses per service
+                let tag = `${page * hitsPerPage + index + 1}`;
+                if (i > 0) {
+                  const alphabeticalIndex = (i + 9).toString(36).toUpperCase();
+                  tag += alphabeticalIndex;
+                }
+
+                return tag;
+              };
+
               markers.push(<SearchHitMarker
-                key={hit.id}
+                // eslint-disable-next-line react/no-array-index-key
+                key={`${hit.id}.${i}`}
```

Comment:
You probably don't want to disable this ESLint warning, since it could cause weird cache invalidation issues in React in the case where addresses get changed around (maybe removed due to additional filters?). I think what you actually want to do is embed something that is unique to the address itself as the key, perhaps the lat/lng?

Reply:
I might be being paranoid, but I changed it to the `id + lat/lng + address_1 + address_2` in case we get services with multiple addresses within the same building